src/translib/request_binder.go

```diff
@@ -42,15 +43,17 @@ func initSchema() {
 }
 
 type requestBinder struct {
-	uri             *string
-	payload         *[]byte
-	opcode          int
-	appRootNodeType *reflect.Type
-	pathTmp         *gnmi.Path
+	uri                *string
+	payload            *[]byte
+	opcode             int
+	appRootNodeType    *reflect.Type
+	pathTmp            *gnmi.Path
+	targetNodePath     *gnmi.Path
+	targetNodeListInst bool
 }
 
 func getRequestBinder(uri *string, payload *[]byte, opcode int, appRootNodeType *reflect.Type) *requestBinder {
-	return &requestBinder{uri, payload, opcode, appRootNodeType, nil}
+	return &requestBinder{uri, payload, opcode, appRootNodeType, nil, nil, false}
```

Comment:
It would be good to fix indentation in this file (and throughout this PR) to 4-space tabs